src/web/mjs/connectors/MangaAy.mjs

```diff
@@ -20,7 +20,19 @@ export default class MangaAy extends Connector {
     }
 
     async _getMangas() {
-        let request = new Request(new URL('/seriler', this.url), this.requestOptions);
+        let mangaList = [];
+        const request = new Request(new URL('/seriler', this.url), this.requestOptions);
+        const data = await this.fetchDOM(request, 'nav > ul > li:last-child > a');
+        const pageCount = parseInt(data[0].href.match(/(\d+)$/)[1]);
+        for(let page = 0; page <= pageCount; page+=100) {
```

Comment:
Cannot confirm that there are 100 mangas per page (i only see 47 overall), but i assume this was tested and confirmed